Acrolinx Batch 4 Edits (#3774)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/includes/configure-proxy.md

---
author: dbradish-microsoft
ms.author: dbradish
ms.date: 08/1/2023
ms.topic: include
ms.custom: devx-track-azurecli
---
If you're unable to connect to an external resource due to a proxy, make sure that you've correctly set the `HTTP_PROXY` and `HTTPS_PROXY` variables in your shell. You need to contact your system administrator
to know what host(s) and port(s) to use for these proxies.

> Linux programs utilize these values, including ones that are used in the install process. 

Reviewer: ```suggestion
The values in these variables are used by many Linux programs, and during the install process.
```
